sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/BUILD

```diff
@@ -256,10 +258,18 @@ _FT_PROVISIONING_BINARIES = {
     ":ft_personalize": "ft_personalize",
 }
 
+config_setting(
+    name = "signer_config",
+    values = {"define": "cert_signer=cloud_kms"},
```

Comment:
`define` is a legacy syntax, we should "Prefer Starlark build settings whenever possible" according to [here](https://bazel.build/docs/configurable-attributes#custom-flags) I believe. I think a `[bool_flag](https://github.com/bazelbuild/bazel-skylib/blob/0d43695bd5026a6dc1433ba455f29551311ebe60/rules/common_settings.bzl#L82)` from Skylib should do the trick.

So this would look something like:
```
load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
...
bool_flag(
  name = "endorse_certs_with_ckms",
  build_setting_default = False,
)

config_setting(
    name = "ckms_cert_endorsement_params",
    flag_values = {":endorse_certs_with_ckms": "true"},
)
...
_FT_PROVISIONING_CMD_ARGS = """
  --elf={sram_ft_individualize}
  --bootstrap={ft_personalize}
""" + FT_PROVISIONING_INPUTS + select({
    ":ckms_cert_endorsement_params": CLOUD_KMS_ENDORSEMENT_PARAMS,
    "//conditions:default": FAKE_ENDORSEMENT_PARAMS,
})
```